dspace-api/src/main/java/org/dspace/eperson/EPersonServiceImpl.java

```diff
@@ -179,45 +211,192 @@ public EPerson create(Context context) throws SQLException, AuthorizeException {
+                                                                                              versionHistory)) {
+                                    version.setePerson(null);
+                                    versionDAO.save(context, version);
+                                }
+                            }
+                            WorkspaceItemService workspaceItemService = ContentServiceFactory.getInstance()
+                                                                        .getWorkspaceItemService();
+                            WorkspaceItem wsi = workspaceItemService.findByItem(context, item);
+
+                            if (null != wsi) {
+                                workspaceItemService.deleteAll(context, wsi);
+                            } else {
+                                // we can do that as dc.provenance still contains
+                                // information about who submitted and who
+                                // archived an item.
+                                item.setSubmitter(null);
+                                itemService.update(context, item);
+                            }
+                        }
+                    } else if (StringUtils.equals(tableName, "cwf_claimtask") && isXmlFramework) {
+                         // Unclaim all XmlWorkflow tasks
+                        XmlWorkflowItemService xmlWorkflowItemService = XmlWorkflowServiceFactory
+                                                                        .getInstance().getXmlWorkflowItemService();
+                        ClaimedTaskService claimedTaskService = XmlWorkflowServiceFactory
+                                                                .getInstance().getClaimedTaskService();
+                        XmlWorkflowService xmlWorkflowService = XmlWorkflowServiceFactory
+                                                                .getInstance().getXmlWorkflowService();
+                        WorkflowRequirementsService workflowRequirementsService = XmlWorkflowServiceFactory
+                                                                       .getInstance().getWorkflowRequirementsService();
+
+                        List<XmlWorkflowItem> xmlWorkflowItems = xmlWorkflowItemService
+                                                                 .findBySubmitter(context, ePerson);
+
+                        for (XmlWorkflowItem xmlWorkflowItem : xmlWorkflowItems) {
+                            ClaimedTask pooledTask = claimedTaskService
+                                                     .findByWorkflowIdAndEPerson(context, xmlWorkflowItem, ePerson);
+                            xmlWorkflowService.deleteClaimedTask(context, xmlWorkflowItem, pooledTask);
+
+                            try {
+                                workflowRequirementsService.removeClaimedUser(context, xmlWorkflowItem,
```

Comment:
What happens if the user is in the pool, will they be reassigned?
What happens if the user is/was the only user in the pool?